src/lib/cli/rule-generator.ts

```diff
@@ -80,7 +80,7 @@ const loadPopulatedTemplate = async (filePath: string, data: Object): Promise<st
     try {
         content = await util.promisify(fs.readFile)(filePath, 'utf8');
     } catch (e) {
-        debug(`Error reading file: ${filePath}`);
+        logger.debug(`Error reading file: ${filePath}`);
```

Comment:
I notice in several places you're using `logger.debug` to log errors. I think `logger.error` would be more appropriate.